### PR TITLE
Change request event tracking

### PR DESCRIPTION
**Pull request by kwasniew**
## About the changes
* adding change request event tracking
* fixing change request process img and description
* fixing types in HTTP client
* renaming HTTP client method to addChange from addChangeRequest
* Fixing IChangeSchema type name

## Discussion points

* tracking added in HTTP client because it's called in many places
* tracking change request on is done at the component level (in HTTP client there's config change request and it's difficult to find what's changing)

### CODE REVIEW

File: frontend/src/component/project/Project/ProjectSettings/ChangeRequestConfiguration/ChangeRequestProcessHelp/ChangeRequestProcessHelp.tsx [hunk 1]
```diff
@@ -31,8 +31,12 @@ export const ChangeRequestProcessHelp: VFC<
                     </Typography>
                 }
             />
-            <IconButton title="Change request process" ref={ref}>
-                <HelpOutline onClick={() => setIsOpen(true)} />
+            <IconButton
+                title="Change request process"
+                ref={ref}
+                onClick={() => setIsOpen(true)}
```
Comment (kwasniew) [inline frontend/src/component/project/Project/ProjectSettings/ChangeRequestConfiguration/ChangeRequestProcessHelp/ChangeRequestProcessHelp.tsx:37]: we were getting react warning here so fixing it

File: frontend/src/component/feature/FeatureStrategy/FeatureStrategyCreate/FeatureStrategyCreate.tsx [hunk 2]
```diff
@@ -41,7 +41,7 @@ export const FeatureStrategyCreate = () => {
     const errors = useFormErrors();
 
     const { addStrategyToFeature, loading } = useFeatureStrategyApi();
-    const { addChangeRequest } = useChangeRequestApi();
+    const { addChange } = useChangeRequestApi();
```
Comment (kwasniew) [inline frontend/src/component/feature/FeatureStrategy/FeatureStrategyCreate/FeatureStrategyCreate.tsx:44]: addChangeRequest was incorrect name

File: frontend/src/component/project/Project/ProjectSettings/ChangeRequestConfiguration/ChangeRequestConfiguration.tsx [hunk 3]
```diff
@@ -256,7 +258,17 @@ export const ChangeRequestConfiguration: VFC = () => {
             </Table>
 
             <Dialogue
-                onClick={() => onConfirm()}
+                onClick={() => {
+                    trackEvent('change_request', {
+                        props: {
+                            eventType: `change request ${
+                                !dialogState.isEnabled ? 'enabled' : 'disabled'
```
Comment (kwasniew) [inline frontend/src/component/project/Project/ProjectSettings/ChangeRequestConfiguration/ChangeRequestConfiguration.tsx:265]: we track enable and disable change requests

File: frontend/src/hooks/api/actions/useChangeRequestApi/useChangeRequestApi.ts [hunk 4]
```diff
@@ -1,6 +1,7 @@
 import useAPI from '../useApi/useApi';
+import { usePlausibleTracker } from '../../../usePlausibleTracker';
 
-export interface IChangeRequestsSchema {
+export interface IChangeSchema {
```
Comment (kwasniew) [inline frontend/src/hooks/api/actions/useChangeRequestApi/useChangeRequestApi.ts:4]: this is schema for a single change

File: frontend/src/hooks/api/actions/useChangeRequestApi/useChangeRequestApi.ts [hunk 5]
```diff
@@ -43,8 +51,14 @@ export const useChangeRequestApi = () => {
     const changeState = async (
         project: string,
         changeRequestId: number,
-        payload: any
+        payload: { state: 'Approved' | 'Applied' | 'Cancelled' | 'In review' }
```
Comment (kwasniew) [inline frontend/src/hooks/api/actions/useChangeRequestApi/useChangeRequestApi.ts:54]: now we get nice autocompletion

File: frontend/src/hooks/usePlausibleTracker.ts [hunk 6]
```diff
@@ -8,7 +8,7 @@ import { EventOptions, PlausibleOptions } from 'plausible-tracker';
  * @see https://plausible.io/docs/custom-event-goals#2-create-a-custom-event-goal-in-your-plausible-analytics-account
  * @example `'download | 'invite' | 'signup'`
  **/
-type CustomEvents = 'invite' | 'upgrade_plan_clicked';
+type CustomEvents = 'invite' | 'upgrade_plan_clicked' | 'change_request';
```
Comment (kwasniew) [inline frontend/src/hooks/usePlausibleTracker.ts:11]: it may not scale in future

**Review by sjaanus — APPROVED** [review]
LGTM!

**Pull request merged**